src/cfnlint/core.py

```diff
@@ -175,10 +178,10 @@ def get_template_rules(filename, args):
         args.mandatory_checks,
     )
 
-    return(template, rules, [])
+    return (template, rules, [])
 
 
-def run_checks(filename, template, rules, regions, mandatory_rules=None):
+def run_checks(filename, regions, runner):
```

Comment:
think this would break existing integrations:
https://github.com/aws-cloudformation/cfn-python-lint/blob/master/docs/getting_started/integration.md

Reply:
+1, let's not change this

Reply:
Reverted in new patch